sdks/open-format/src/core/sdk.ts

```diff
@@ -54,14 +55,15 @@ export class OpenFormatSDK extends BaseContract {
    * @param {string} nft.url - storage URL
    * @returns transaction
    */
-  async deploy(nft: NFTMetadata) {
+  async deploy(nft: NFTMetadata, transactionArgs?: Transaction) {
     invariant(this.signer, 'No signer set, cannot deploy');
 
     await this.checkNetworksMatch();
 
     const tx = await contract.deploy({
       signer: this.signer,
       nft,
+      ...(transactionArgs && { transactionArgs }),
```

Comment:
```suggestion
      transactionArgs,
```

LGTM. But you don't need this check here, `transactionArgs` is optional in both functions.